pkg/scheduler/framework/plugins/volumezone/volume_zone.go

```diff
@@ -33,13 +35,18 @@ import (
 	"k8s.io/klog/v2"
 	"k8s.io/kubernetes/pkg/scheduler/framework"
 	"k8s.io/kubernetes/pkg/scheduler/framework/plugins/names"
+	"k8s.io/kubernetes/pkg/scheduler/util"
 )
 
 // VolumeZone is a plugin that checks volume zone.
 type VolumeZone struct {
 	pvLister  corelisters.PersistentVolumeLister
 	pvcLister corelisters.PersistentVolumeClaimLister
 	scLister  storagelisters.StorageClassLister
+
+	// logger is only meant to be used by background activities which don't
+	// have some other logger in their parent callstack.
+	logger klog.Logger
```

Comment:
I don't think this is needed? It seems we passed all logger from the caller. Can we remove this and added it back in a later pr if it's really needed?

Reply:
Sorry, this is my mistake. Removed it